README.md

```diff
@@ -9,7 +10,7 @@ CircuitPython supports [more than 200 different boards](https://circuitpython.or
 
 ![LILYGO T8](docs/T8.jpg)
 
-The planned adapterplate adds a 3-way button for input, place to hold the little 350 mAh battery and interface for I2C with regular 2.54mm 4pin JST XH as well as the small Stemma QT 1mm JST SH connector (QWIIC).
+The planned adapterplate adds a 3-way button for input, place to hold the little 500 mAh battery and interface for I2C with regular 2.54mm 4pin JST XH as well as the small Stemma QT 1mm JST SH connector (QWIIC).
```

Comment:
might be 530 or 600 mAh in the 20mm between the 2 DIL connectors

Reply:
A time estimate and some experimental values should be added after some time.